packages/frontend/src/lib/RecipeCardTags.svelte

```diff
@@ -0,0 +1,167 @@
+<script lang="ts">
+import type { Recipe } from '@shared/src/models/IRecipe';
+import Badge from './Badge.svelte';
+import { onDestroy, onMount } from 'svelte';
+
+interface Props {
+  recipe: Recipe;
+}
+
+let { recipe }: Props = $props();
+
+const useCases = ['natural-language-processing', 'audio', 'computer-vision'];
+const languages = ['java', 'javascript', 'python'];
+const frameworks = ['langchain', 'langchain4j', 'quarkus', 'react', 'streamlit', 'vectordb'];
+const tools = ['none', 'llama-cpp', 'whisper-cpp'];
+
+const tags: string[] = [
+  ...recipe.categories,
+  ...(recipe.backend !== undefined ? [recipe.backend] : []),
+  ...(recipe.frameworks ?? []),
+  ...(recipe.languages ?? []),
+];
```

Comment:
Can we move this to a dedicated constants / tags-constant / please find a nice name file ? as those are constants ? (user uppercase for constants